### PR TITLE
Add vehicle, operatingAirline and pnr fields to Leg

### PR DESCRIPTION
**Pull request by vborszeki**

### CODE REVIEW

File: src/booking/dataloaders/__tests__/__snapshots__/ApiSanitizer.test.js.snap [hunk 1]
```diff
@@ -192,7 +228,19 @@ Object {
...
         "isReturn": false,
+        "operatingAirline": Object {
+          "iata": "LO",
+          "name": "LOT Polish Airlines",
+        },
+        "pnr": "test",
         "recheckRequired": false,
+        "vehicle": Object {
+          "manufacturer": null,
+          "meal": null,
```
Comment (kantord) [inline src/booking/dataloaders/__tests__/__snapshots__/ApiSanitizer.test.js.snap]: It'd be nice to have an example of an actual airplane model in the tests

File: src/flight/types/outputs/OperatingAirline.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import { GraphQLObjectType, GraphQLString } from 'graphql';
+
+import type { OperatingAirline } from '../../Flight';
+
+export default new GraphQLObjectType({
+  name: 'OperatingAirline',
+  fields: {
+    iata: {
+      type: GraphQLString,
```
Comment (kantord) [inline src/flight/types/outputs/OperatingAirline.js]: It'd be nice to have a `description` field for some of the GraphQL field, for example `iata` is not immediately clear for me

File: src/flight/Flight.js [hunk 3]
```diff
@@ -54,3 +57,13 @@ export type Airline = {|
   code: string,
   isLowCost: boolean,
 |};
+
+export type OperatingAirline = {|
+  iata: string,
```
Comment (tbergq) [inline src/flight/Flight.js]: could you make these types read-only

File: src/flight/Flight.js [hunk 4]
```diff
@@ -54,3 +57,13 @@ export type Airline = {|
...
+  name: string,
+|};
+
+export type Vehicle = {|
+  manufacturer: string,
+  model: string,
```
Comment (tbergq) [inline src/flight/Flight.js]: also these

**Pull request merged**